prql-compiler/src/semantic/context.rs

```diff
@@ -196,14 +197,20 @@ impl Context {
             }
         }
 
+        // 1535: I think the thing we need to do is allow this lookup to create
+        // a new module!
```

Comment:
A little further down, but generally, correct.

Reply:
But once we hit [this line](https://github.com/PRQL/prql/pull/2431/files#diff-54f0771b2a1409b9bf65752bfcb28d90ff4681d8c8fff92af3581c284cbef0d4R214) `0 => Err(format!("Unknown name {ident}")),`, we raise. 

So presumably we need to add a new module into the root before we hit [this section](https://github.com/PRQL/prql/pull/2431/files#diff-54f0771b2a1409b9bf65752bfcb28d90ff4681d8c8fff92af3581c284cbef0d4R218-R225) — is that correct?